spinta/manifests/tabular/helpers.py

```diff
@@ -1550,6 +1550,9 @@ def _property_to_tabular(
     if prop.access < access:
         return
 
+    remember_unique_state = prop.dtype.unique
+    if len(prop.model.external.pkeys) == 1:
+        prop.dtype.unique = False
```

Comment:
Please move this into `dtype.get_type_repr`, because this method is responsible for producing correct  type string. Also it would be nice, to move `get_type_repr` out of `dtype`, because, this representation only needed for tabular manifests, so it should be part of tabular manifest, not a high level method.

Reply:
Same issue here: https://github.com/atviriduomenys/spinta/pull/343#discussion_r1176129421